Source/InMobiAdapter.swift

```diff
@@ -65,8 +77,13 @@ final class InMobiAdapter: NSObject, PartnerAdapter {
     /// - parameter request: Information about the ad load request.
     /// - parameter completion: Closure to be performed with the fetched info.
     func fetchBidderInformation(request: PreBidRequest, completion: @escaping ([String : String]?) -> Void) {
-        // InMobi does not currently provide any bidding token
-        completion(nil)
+        log(.fetchBidderInfoStarted(request))
+        guard let bidToken = IMSdk.getToken() else {
+            log(.fetchBidderInfoFailed(request, error: InMobiAdapterError.failedToProvideBidToken))
```

Comment:
we could simplify removing the need for a custom error.
```suggestion
            log(.fetchBidderInfoFailed(request, error: error(.prebidFailureUnknown, description: "Failed to provide bid token.")))
```